pkg/asset/internal/templates.go

```diff
@@ -871,24 +873,33 @@ metadata:
     kubernetes.io/name: "CoreDNS"
     kubernetes.io/cluster-service: "true"
 spec:
-  # replicas: not specified here:
-  # 1. In order to make Addon Manager do not reconcile this replicas parameter.
-  # 2. Default is 1.
-  # 3. Will be tuned in real time if DNS horizontal auto-scaling is turned on.
+  replicas : 2
   strategy:
     type: RollingUpdate
     rollingUpdate:
       maxUnavailable: 1
   selector:
     matchLabels:
-      k8s-app: kube-dns
+      k8s-app: coredns
```

Comment:
FWIW, https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/addons/dns/manifests.go#L224. 

I think that is to align it with the labels and the kubernetes DNS service. Do we want to stick to the same?

Reply:
Not sure of the upside or downside of keeping it as `kube-dns`

Reply:
looks like it was changed to coredns but got changed back to kubedns in this PR https://github.com/kubernetes-sigs/bootkube/pull/1038 - the particular commit https://github.com/kubernetes-sigs/bootkube/commit/daf844ac8e9aea459688bb9611aaf937fd44c027

Reply:
I have a feeling we should leave as `kube-dns` but we can go with this in the mean time.